Card Component/card.css

```diff
@@ -0,0 +1,146 @@
+*{
+    padding: 0px;
+    margin: 0px;
+    font-size: 62.5%;
+    box-sizing: border-box;
+}
+
+body{
+  
+}
+
+.main{
+    display: flex;
+    flex-direction: column;
+    gap : 2rem;
+    justify-content: center;
+    align-items: center;
+    background-color: #686672;
+}
+
+
+.card-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.card{
+
+    background-color: white;
+    border: none;
+    margin-left:0px ;
```

Comment:
you may mention only 0 instead of 0px

Reply:
Thanks for suggestion, I will change it to 0.